src/main/java/com/takipi/oss/storage/TakipiStorageMain.java

```diff
@@ -61,6 +62,7 @@ public void run(TakipiStorageConfiguration configuration, Environment environmen
 
         Filesystem filesystem = configureFilesystem(configuration, environment);
         TakipiStorageConfiguration.Multifetch multifetchConfig = configuration.getMultifetch();
+        MultiFetcher.setMaxCacheSize(multifetchConfig.getMaxCacheSize());
```

Comment:
Pass the cache size through the constructor of `JsonMultiFetchStorageResource`. Just the way we do for the concurrency level

Reply:
done